.travis.yml

```diff
@@ -4,14 +4,29 @@ matrix:
   fast_finish: true
   include:
   - name: "Build BurstJS"
+    os: "linux"
     node_js: "10"
     env: JOB=burstjs
   - name: "Build Web Wallet"
+    os: "linux"
     node_js: "10"
     env: JOB=web
+  - name: "Build Desktop Wallet"
+    if: branch = test-deploy
+    os: linux
+    node_js: "10"
+    services:
+      - docker
+    env: JOB=desktop
+
```

Comment:
This is gonna be awesome.

Reply:
Need to explore that stuff more. My goal is to make the executable available as artefacts on github and list the app under https://electronjs.org/apps